setup.py

```diff
@@ -53,14 +53,16 @@
 import sys
 
 MIN_TORCH_JITTABLE_VERSION = "1.3.0"
+MAX_TORCH_JITTABLE_VERSION = "1.5.0"
```

Comment:
Move these two constants into brevitas/__init__.py where they are also used, so that we have a single source of truth. __init__.py is not importing anything new compared to setup.py so it should be fine

Reply:
Also, for coherence it should be MAX_TORCH_JITTABLE_VERSION = 1.4.0, since the MIN one is inclusive, and then adjust comparison operators in init and setup accordingly

Reply:
In  **init.py** there's the import of packaging that breaks everything. Considering the workaround we have in setup.py for having it installed automatically, I would prefer duplicating the two constants for now and look for a cleaner solution in a second moment

Reply:
Just move the import inside the run() methods and then pass the value to the template function.

Reply:
Actually we can just define a bool is_ste_jittable in \_\_init_\_.py and then import it into setup.py from within the run methods.

Reply:
Importing within the run methods causes all the code in __init__ to be executed, included the search for the "_C" file. But at setup-time, the _C module has yet to be created, so it fails.

Reply:
That's a good point, let's bool the bool in brevitas.config and import it into both init and setup. We need to setup a separate pull to move docstrings out of config first though, and into brevitas' init, or we will have another setup time import error.